lib/transport_web/controllers/page_controller.ex

```diff
@@ -17,4 +18,29 @@ defmodule TransportWeb.PageController do
       render conn, "shortlist.html"
   end
 
+  def user_organizations(conn, _) do
```

Comment:
Apparently it is more idiomatic to add blank spaces https://github.com/christopheradams/elixir_style_guide#def-spacing